front/src/actions/dashboard/boxes/weather.js

```diff
@@ -38,7 +48,7 @@ function createActions(store) {
         });
         weather.days.map(day => {
           day.weather_icon = translateWeatherToFeIcon(day.weather);
-          day.datetime_beautiful = dayjs(day.datetime).format('D MMM');
+          day.datetime_beautiful = DAYS_OF_WEEKS[dayjs(day.datetime).day()];
```

Comment:
Why not use `dayjs(day.datetime).format('dddd');` ? Much more simple :) 

Documentation: https://day.js.org/docs/en/display/format

Reply:
Yes :(
I had not seen it